integration-tests/tests/models/single-legacy.spec.ts

```diff
@@ -443,7 +443,7 @@ describe('other', () => {
 
       let cdnMetadataResult = await fetchMetadataFromCDN();
       expect(cdnMetadataResult.status).toEqual(200);
-      expect(cdnMetadataResult.body).toEqual([{ c0: 1 }]);
+      expect(cdnMetadataResult.body).toEqual({ c0: 1 });
```

Comment:
Every past call we registered was a Mesh artifact and it's expected to be an object, not an array, so it's not a breaking change.